README.md

```diff
@@ -1,70 +1,159 @@
-# Vaadin platform
+<div align="center">
 
-[Vaadin platform](https://vaadin.com/) consists of a set of web components, a Java web framework called Vaadin Flow, configurable themes, tools and a set of opinionated app starters. Platform releases gather all products in to a single package every quarter. The product versions included in a platform release are not necessarily the latest released versions, but are hand picked so that they work well together.
+![Vaadin](vaadin-banner.svg)
 
-Visit [vaadin.com](https://vaadin.com/) to learn more about Vaadin.
+Vaadin is a web app development platform for Java. It helps you build reliable web apps with great UX faster than before.
```

Comment:
Let's consider adding a line break here to make the description look more readable.

### Before

![Screenshot 2021-11-22 at 12 23 47](https://user-images.githubusercontent.com/10589913/142844859-94bfcf81-f6ba-4a13-b73d-f2826e553826.png)

### After

![Screenshot 2021-11-22 at 12 24 19](https://user-images.githubusercontent.com/10589913/142844900-9bdc15b3-751e-4ec9-be37-128e0b34ba1d.png)

Reply:
Yeah, that looks nicer at least on large viewport sizes. I suppose that’s more common than a small viewport

![Screen Shot 2021-11-22 at 12 28 37](https://user-images.githubusercontent.com/66382/142845440-16c20c69-eca7-424a-8411-3da80a7ef4a1.png)

Reply:
Thanks, that makes sense. Let's keep it then.